### PR TITLE
API: add login username in SendLoginLogCommand

### PR DESCRIPTION
**Pull request by AgnesToulet**
**What this PR does / why we need it**:
Add the username used for Grafana login in the `SendLoginLogCommand`. It's useful information when the login fails with a "user is disabled", "user lockout" or "invalid username and password" error. 

**Special notes for reviewers:**
I'm not sure about the labels I should use for this to be in 7.3.

### CODE REVIEW

**Review by xlson — APPROVED** [review]

File: pkg/api/login_test.go [hunk 1]
```diff
@@ -687,6 +687,7 @@ func TestLoginPostSendLoginLog(t *testing.T) {
 
 			cmd := testReceiver.cmd
 			assert.Equal(t, c.cmd.LogAction, cmd.LogAction)
+			assert.Equal(t, c.cmd.LoginUsername, "admin")
```
Comment (aknuds1) [inline pkg/api/login_test.go:690]: I think you want to do the following? The second argument to assert.Equal is the expectation, and the third is what you got. What you got should be cmd.LoginUsername right?
```suggestion
			assert.Equal(t, "admin", cmd.LoginUsername)
```

**Review by aknuds1 — CHANGES_REQUESTED** [review]
I think I spotted a bug in the test.

File: pkg/api/login_test.go (hunk 1 above)
Comment (AgnesToulet) [reply pkg/api/login_test.go:690]: Yes exactly, I just fixed it, thanks! I'm still not used to put the expected value first.

**Comment by AgnesToulet** [comment]
@aknuds1 Thanks, I fixed this. Do you know if the labels are good to have this in 7.3?

**Review by aknuds1 — APPROVED** [review]
LGTM

**Comment by aknuds1** [comment]
> @aknuds1 Thanks, I fixed this. Do you know if the labels are good to have this in 7.3?

Yes, looks correct!

**Pull request merged**